test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -239,7 +242,9 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 			podLabels := map[string]string{
 				"test": "test-daemonset",
 			}
-			nodeSelector := map[string]string{}
+			nodeSelector := map[string]string{
+				serialconfig.MultiNUMALabel: "2",
```

Comment:
do we need this change in this patch?

Reply:
Yes. In case there are none multi numa machines in the cluster (usually developer use-case) the daemonset will place a pod in there. 

Usually, the single NUMA machines are not strong enough and they got filtered by the `NodeResourceFit` plugin which in turn leads to test failure because we're looking for the `NodeTopologyMatch` error message and are unable to find it.

Reply:
Uhm, makes sense but please split this change in its own commit (please copy/paste [a possibly enhanced version of] this explanation there)

Reply:
Done.